### DISCUSSION TITLE
DietPi-Software | RepetierServer

### DISCUSSION THREAD

**Discussion by oradke** [MichaIng/DietPi#6236] [Software Request]
#### Formal software information
- Software title | RepetierServer
- Short description | 3D print server
- Official URL (if available) | https://www.repetier-server.com
- GitHub URL (if available) | n/A
- (Official) install documentation | https://www.repetier-server.com/manuals/1.4/index.html

#### Are there similar/alternative software titles available with DietPi-Software?

- Octoprint

#### What makes your requested software better than the above solutions, if available?

- Lots of features
- Good touchscreen interface
- Webcam itegrated
- Clients available for PCs (https://www.repetier-server.com/repetier-server-monitor-2/) and smartphones 

#### How can DietPi make the installation easier or compatible, than following the install instructions or do APT installation, if available?

- Repetierserver comes in a handy package, but touchscreen and webcam support is tedious to install. There's a ready-to-burn image available, but it's not as nicely to manage as a dietpi installation and does not easily allow additional software to run.

#### Can you provide the installation steps that you would suggest DietPi-Software to do?

1. install Repetierserver
2. install touchscreen interface, install and configure chromium kiosk mode 
3. install webcam (mjpg-streamer etc. https://www.repetier-server.com/setting-webcam-repetier-server-linux/)

#### Are you willing to help maintaining the software installation, e.g. in case of needed setup changes due to updates etc.? This is not needed, but could speed up our decision to implement it, as man power is always a topic :wink:.

- Sure! I'm not a Linux expert, but I can see when things break and do some troubleshooting and testing.

**Comment by Joulinar** [comment]
Converted to discussion, so others please vote for this software implementation via ⬆️ button at first post.
